docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -36,40 +49,90 @@ Start by fetching Chipyard's sources. Run:
+
+.. code-block:: shell
+
+    conda activate chipyard
+
+We recommend that you add this "activate" command your ``.bashrc`` (or other environment setup file).
+
+Fetch Chipyard Sources
+~~~~~~~~~~~~~~~~~~~~~~
+
+To fetch all Chipyard sources, run the following:
+
+.. code-block:: shell
+
     ./scripts/init-submodules-no-riscv-tools.sh
 
 This will initialize and checkout all of the necessary git submodules.
 This will also validate that you are on a tagged branch, otherwise it will prompt for confirmation.
 
 When updating Chipyard to a new version, you will also want to rerun this script to update the submodules.
-Using git directly will try to initialize all submodules; this is not recommended unless you expressly desire this behavior.
-
-.. _build-toolchains:
+Using ``git`` directly will try to initialize all submodules; this is not recommended unless you expressly desire this behavior.
 
-Building a Toolchain
+Obtaining a Toolchain
 ------------------------
 
-The `toolchains` directory contains toolchains that include a cross-compiler toolchain, frontend server, and proxy kernel, which you will need in order to compile code to RISC-V instructions and run them on your design.
-Currently there are two toolchains, one for normal RISC-V programs, and another for Hwacha (``esp-tools``).
-For custom installations, Each tool within the toolchains contains individual installation procedures within its README file.
+Currently there are two toolchains, one for normal RISC-V programs, and another for Hwacha/Gemmini (``esp-tools``).
 To get a basic installation (which is the only thing needed for most Chipyard use-cases), just the following steps are necessary.
-This will take about 20-30 minutes. You can expedite the process by setting a ``make`` environment variable to use parallel cores: ``export MAKEFLAGS=-j8``.
 
 .. code-block:: shell
 
-    ./scripts/build-toolchains.sh riscv-tools # for a normal risc-v toolchain
+    conda install -c ucb-bar ucb-bar-riscv-tools # for a normal risc-v toolchain
+
+.. Note:: If you are planning to use the Hwacha vector unit, or other RoCC-based accelerators, you should obtain the esp-tools toolchain by adding the ``ucb-bar-esp-tools`` argument to the command above.
+
+Once the command is run, the ``PATH``, ``RISCV``, and ``LD_LIBRARY_PATH`` environment variables will be set properly.
 
-.. Note:: If you are planning to use the Hwacha vector unit, or other RoCC-based accelerators, you should build the esp-tools toolchain by adding the ``esp-tools`` argument to the script above.
-  If you are running on an Amazon Web Services EC2 instance, intending to use FireSim, you can also use the ``--ec2fast`` flag for an expedited installation of a pre-compiled toolchain.
+.. Note:: If you are a power user and would like to build your own toolchain, you can refer to the https://github.com/ucb-bar/riscv-tools-feedstock and https://github.com/ucb-bar/esp-tools-feedstock repositories on how to build a toolchain yourself.
```

Comment:
Didn't we flatten riscv-tools and esp-tools into chipyard in the first place to make it easier to bump those projects? This adds back that level of indirection.

In fact, doesn't this add two levels of indirection? Now we additionally have to track commits for riscv-tools-feedstock and riscv-tools.

Reply:
Yes. But for the most part, the main item that is being rebuilt is Spike (for Gemmini folks). That can be rebuilt separately. In the future we will focus on splitting up things even more.